backend-api/template.yaml

```diff
@@ -259,6 +259,13 @@ Resources:
       Environment:
         Variables:
           CLIENT_REGISTRY_SECRET_NAME: !Sub ${Environment}/clientRegistry
+      VpcConfig:
```

Comment:
Please may we write an infra test that checks all lambdas have a VpcConfig with 3 SubnetIds and a Security group attached?

Reply:
Done, but please check if ok :)